core/modules/savers/put.js

```diff
@@ -21,18 +21,20 @@ Select the appropriate saver module and set it up
 var PutSaver = function(wiki) {
 	this.wiki = wiki;
 	var self = this;
+	var uri = encodeURI(document.location.protocol + "//" + document.location.hostname + ":" + document.location.port + document.location.pathname);
```

Comment:
This approach was introduced in f51f6bf774d7acf3304f516e2ff0af4f4e05fff1 because "We don’t want the location hash" - would it perhaps be more readable to use `document.location.toString().split("#")[0]`?